### PR TITLE
Escaping special chars for cli command and unused removing flags

### PR DESCRIPTION
**Pull request by jak-atx**
removing password and vic-appliance-endpoint flags

escaping () and whitespace between words for CLI

Fixes #

### CODE REVIEW

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts [hunk 1]
```diff
@@ -253,14 +254,17 @@ export class SummaryComponent implements OnInit {
...
     if (results['operations']['opsPassword']) {
-      results['operations']['opsPassword'] = '*';
+      delete results['operations']['opsPassword'];
     }
 
     return results;
   }
 
+  escapeSpecialCharsForCLI(text) {
+    return text.replace(/[\]{}()*+?,\\^$|#\s]/g, '\\$&');
```
Comment (jooskim) [inline h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts]: Does `\$*` translate to some special var? And how did you come up with those special characters?

Comment (jak-atx) [reply h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts]: $& is a "replacement" (placeholder for something to be substituted in) for the full match (all the matched text)

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts [hunk 2]
```diff
@@ -253,14 +254,17 @@ export class SummaryComponent implements OnInit {
...
     if (results['operations']['opsPassword']) {
-      results['operations']['opsPassword'] = '*';
+      delete results['operations']['opsPassword'];
     }
 
     return results;
   }
 
+  escapeSpecialCharsForCLI(text) {
+    return text.replace(/([() ])/g, '\\$&');
```
Comment (jak-atx) [inline h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts]: For now only including what we know we need to escape which is () and whitespace in between words.

**Comment by jak-atx** [comment]
slack lgtm from @jooskim :)

**Pull request merged**